.buildkite/hooks/pre-command

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+##  This script prepares the secret context
+##
+##  NOTE: *_SECRET or *_TOKEN env variables are masked, hence if you'd like to avoid any
+##        surprises please use the suffix _SECRET or _TOKEN for those values that contain
+##        any sensitive data. Buildkite can mask those values automatically
+
+set -eo pipefail
+
+# To help with testing the GPG signing
+BUILDKITE_TOKEN_SECRET=$(vault kv get -field=buildkite_token kv/ci-shared/observability-ci/buildkite-read-build-access)
```

Comment:
There is an ongoing task to move away from BK API Tokens and use some opinionated contracts so we can skip this API call for the sake of simplicity. Release team is working on it